package.json

```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "^14.1.0",
-    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-import": "^2.20.2",
```

Comment:
Did you intentionally switch from yarn/yarn.lock -> npm/package-lock.json?

Reply:
yes. i thought for a public project sticking to the default was better than picking favourites.